[2025-04-07T20:50:09.096+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-07T20:50:09.116+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tmdb_pipeline_streamlit.load_to_postgres manual__2025-04-07T20:47:17.857279+00:00 [queued]>
[2025-04-07T20:50:09.123+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tmdb_pipeline_streamlit.load_to_postgres manual__2025-04-07T20:47:17.857279+00:00 [queued]>
[2025-04-07T20:50:09.124+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-04-07T20:50:09.136+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_to_postgres> on 2025-04-07 20:47:17.857279+00:00
[2025-04-07T20:50:09.144+0000] {standard_task_runner.py:63} INFO - Started process 274 to run task
[2025-04-07T20:50:09.147+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'tmdb_pipeline_streamlit', 'load_to_postgres', 'manual__2025-04-07T20:47:17.857279+00:00', '--job-id', '580', '--raw', '--subdir', 'DAGS_FOLDER/tmdb_pipeline.py', '--cfg-path', '/tmp/tmp9ojp51rj']
[2025-04-07T20:50:09.150+0000] {standard_task_runner.py:91} INFO - Job 580: Subtask load_to_postgres
[2025-04-07T20:50:09.197+0000] {task_command.py:426} INFO - Running <TaskInstance: tmdb_pipeline_streamlit.load_to_postgres manual__2025-04-07T20:47:17.857279+00:00 [running]> on host 0e4950d3ea5b
[2025-04-07T20:50:09.296+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tmdb_pipeline_streamlit' AIRFLOW_CTX_TASK_ID='load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-04-07T20:47:17.857279+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-07T20:47:17.857279+00:00'
[2025-04-07T20:50:09.298+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-07T20:50:09.325+0000] {tmdb_pipeline.py:1915} ERROR - No se encontró el archivo de datos procesados: None
[2025-04-07T20:50:09.349+0000] {tmdb_pipeline.py:1925} INFO - Leyendo 20 registros procesados para cargar en PostgreSQL
[2025-04-07T20:50:09.356+0000] {tmdb_pipeline.py:1498} INFO - Tablas creadas o verificadas correctamente en PostgreSQL
[2025-04-07T20:50:09.359+0000] {tmdb_pipeline.py:1941} INFO - Conexión establecida con PostgreSQL
[2025-04-07T20:50:09.392+0000] {tmdb_pipeline.py:2093} INFO - Total de películas en la base de datos: 62
[2025-04-07T20:50:09.393+0000] {tmdb_pipeline.py:2094} INFO - Insertadas 0 nuevas películas en esta ejecución
[2025-04-07T20:50:09.393+0000] {tmdb_pipeline.py:2100} INFO - Datos cargados correctamente en PostgreSQL
[2025-04-07T20:50:09.394+0000] {python.py:237} INFO - Done. Returned value was: 20
[2025-04-07T20:50:09.395+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-07T20:50:09.415+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=tmdb_pipeline_streamlit, task_id=load_to_postgres, execution_date=20250407T204717, start_date=20250407T205009, end_date=20250407T205009
[2025-04-07T20:50:09.450+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-07T20:50:09.476+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-07T20:50:09.477+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
